### PR TITLE
Added types for ApprovalFlag, SetIndex, VoterInfo

### PR DESCRIPTION
**Pull request by Stefie**
To address additions in https://github.com/paritytech/substrate/commit/840456893df7170c5bed55cb565ead678339b43b#diff-38731d528202a093d94848de6fbc6097

### CODE REVIEW

File: packages/types/README.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ These custom types implement specific types that are found as part of the Substr
 | **Types** | |
 | --- | --- |
 | [[Amount]] | The Substrate Amount representation as a [[Balance]] |
+| [[ApprovalIndex]] | Approval index, implemented as a [[U32]] |
```
Comment (jacogr) [inline packages/types/README.md]: ApprovalFlag

**Comment by Stefie** [comment]
`VoterInfo` is wrong, I was too quick here.

**Comment by Stefie** [comment]
How do I convince codeclimate, that `export default class SomeClass extends Struct {` is not duplicate code?

**Comment by jacogr** [comment]
Ignore it. Always look, but some are just false flags. So a guide rather than a must-pass.

**Pull request merged**